runtime/gc_structs/ConstantPoolObjectSlotIterator.cpp

```diff
@@ -73,6 +73,13 @@ GC_ConstantPoolObjectSlotIterator::nextSlot()
 		if(slotType == J9CPTYPE_METHODHANDLE) {
 			return &(((J9RAMMethodHandleRef *) slotPtr)->methodHandle);
 		}
+		if(slotType == J9CPTYPE_CONSTANT_DYNAMIC) {
+			if(((J9RAMConstantDynamicRef *) slotPtr)->value != NULL) {
```

Comment:
Would you please put NULL to the left side:
if(NULL != ((J9RAMConstantDynamicRef *) slotPtr)->value) {